packages/documentation-site/patternfly-docs/content/get-started/about-patternfly.md

```diff
@@ -0,0 +1,363 @@
+      </CardBody>
+      <Divider />
+      <CardFooter>
+       [Visit our GitHub discussions board <ExternalLinkAltIcon />](https://github.com/orgs/patternfly/discussions/categories/feature-requests)
+      </CardFooter>
+    </Card>
+  </Grid>
+
+<br />
+<br />
+
+#### New communities
+Sometimes Flyers branch out and build groups of their own, creating new communities. While these communities are separate from PatternFly and not supported as part of our design system, we still love seeing our Flyers turning their visions into actions.
+
+- **[PatternFly Elements](https://patternflyelements.org):** A community created by web-based developers at Red Hat, focused on creating web components for use across Red Hat's sites and SaaS products. It offers theming options for your own brand library.
+
+- **[PatternFly Kotlin](https://github.com/patternfly-kotlin/patternfly-kotlin):** A Kotlin implementation of PatternFly based on fritz2, targeting Kotlin/JS. The goal of this project is to provide all PatternFly components in Kotlin, matching the reactive nature of fritz2. The components use stores, handlers, and other elements from the fritz2 API. For a quick overview, [view the PatternFly Kotlin showcase.](https://patternfly-kotlin.github.io/patternfly-kotlin-showcase/#home)
+
+- **[Ansible Component Guide and Sketch Library](https://www.sketch.com/s/6ccbd710-267d-4b69-9dae-bc19e1551056):** An additional resource that designers can use that is built on top of existing PatternFly components. It is a rapid mockup prototyping tool that can be used to quickly put together repeatable design patterns and layouts across projects. This is specific to Ansible, but many of the components are generalized and can fit many product use cases.
+
+- **[PatternFly for Yew](https://github.com/patternfly-yew/patternfly-yew)**: Provides PatternFly components for Yew—a Rust based framework for creating web applications that can run in the browser using WASM. The project aims to create Yew components for all components and concepts found in PatternFly. For a quick demo and starter template, [see the PatternFly Yew Quickstart.](https://github.com/patternfly-yew/patternfly-yew-quickstart)
+
+## Where do I start?
+
+Now that you've been introduced to PatternFly, you're ready to start designing or start developing your product. Looking to get involved in the behind-the-scenes work? Check out our contribution guidelines for more instruction.
+
+<br />
+<Grid sm={12} md={4} hasGutter>
+<Button size="lg" variant="secondary" component="a" href="/get-started/design"> Start designing </Button>
+<Button size="lg" variant="secondary" component="a" href="/get-started/develop"> Start developing </Button>
+<Button size="lg" variant="secondary" component="a" href="/get-started/contribute/contributing-to-patternfly"> Contribute to PatternFly</Button>
```

Comment:
```suggestion
<Button size="lg" variant="secondary" component="a" href="/get-started/contribute"> Contribute to PatternFly</Button>
```
will need to add followup to change this, if https://github.com/patternfly/patternfly-org/pull/3926 closes